languages/python/oso/polar/data.py

```diff
@@ -0,0 +1,78 @@
+# New interface for an adapter.
+# Only does things needed for authorized_resources and authorized_query
+# TODO: Authorized session/database
+
+# todo: build a filter object instead of passing the json through
+# that way we can parse everything to python.
+class Filter:
```

Comment:
Would we go further adding more structure kind of like what @gw000 added in the Ruby lib?

Something like:

`Filter`
`Relations`
`Condition`

`Filter.to_query => Query`
`Filter.to_base_query => Query`
`Relations.add_to_query(query) => Query`
`Condition.add_to_query(query) => Query`
`Filter.add_condition(query)`

```
class Filter:
    def to_query(self):
        # make query w/ from
        query = self.to_base_query()

        
        query = self.relations.add_to_query(query)
        query = reduce(lambda q, cond: cond.add_to_query(q), self.conditions)

        return query
```

(meant to be purely illustrative)